docs/userGuide/plugins/copyCode.mbdf

```diff
@@ -0,0 +1,20 @@
+#### `copyCode`: Enabling code-blocks to be copied
```

Comment:
I feel like the name `copyCode` might not be explicit enough. What do you think of `codeBlockCopyButtons`?

Reply:
I don't have any objection to it. I just tried to make it short, but you're right - it doesn't convey the entire purpose. Will make the change 👍